projectile.el

```diff
@@ -789,12 +805,6 @@ The current directory is assumed to be the project's root otherwise."
             (error "You're not in a project")
           default-directory))))
 
-(defun projectile-file-truename (file-name)
```

Comment:
I'd really have the `if` check just here and use this wrapper everywhere. A self-contained change is the best possible change.